python/lsst/validate/drp/calcSrd.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+import math
+
+import numpy as np
+import scipy.stats
+
+import lsst.pipe.base as pipeBase
+
+def calcPA1(groupView, magKey):
+    """Calculate the photometric repeatability of measurements across a set of observations.
```

Comment:
This docstring doesn't follow the standards as outlined at https://confluence.lsstcorp.org/display/LDMDG/Documentation+Standards#DocumentationStandards-FormattingoftheDocstringText. (Of course, I understand that those standards are up for debate and may be changed or replaced in the future.)

...that aside, it is a model of clarity & usefulness. Thanks!